app/assets/stylesheets/application.scss

```diff
@@ -16,4 +16,51 @@
  */
 
 //= require globals
+//= require select2
 @import 'bootswatch_theme';
+
+.card-info {
```

Comment:
I think that we have already discussed not to write any css in application.scss file